azkaban-common/src/test/java/azkaban/user/FileWatcherTest.java

```diff
@@ -57,6 +72,13 @@ public void tearDown() throws Exception {
     if (this.fileWatcher != null) {
       this.fileWatcher.close();
     }
+
+    // Cleanup directories
```

Comment:
How about using a `TemporaryFolder` `@Rule` for the workDir to simplify code?

For example here:

https://github.com/azkaban/azkaban/blob/7efe67b0602fdc65f18fc0e2f91e14a57c2d845e/azkaban-exec-server/src/test/java/azkaban/execapp/event/LocalFlowWatcherTest.java#L31-L32

Reply:
Thanks. I never knew about this being a Java n00b.
I have updated the test cases.